debian/changelog

```diff
@@ -1,1645 +0,0 @@
-felix (3.0.0~__STREAM__) __STREAM__; urgency=low
```

Comment:
Did you mean to remove this file?  I think it's required by the v3.0 build process.